src/components/application_manager/src/rpc_service_impl.cc

```diff
@@ -535,13 +559,36 @@ void RPCServiceImpl::SendMessageToMobile(
                (*message)[jhs::S_PARAMS][jhs::S_MESSAGE_TYPE].asInt())) {
     allow_unknown_parameters = false;
   }
+
+  const auto api_function_id = static_cast<mobile_apis::FunctionID::eType>(
+      (*message)[strings::params][strings::function_id].asUInt());
+
+  if (app) {
+    mobile_so_factory().attachSchema(*message, false, app->msg_version());
```

Comment:
This code block is resolving a separate issue from #2997 which I believe would need more consideration. Changes to this file should be removed from this PR

Reply:
@jacobkeeler fixed in 986bcf07dafae78bff53a2be8a14998b251ef955